### PR TITLE
Screenshot and build badge

### PR DESCRIPTION
**Pull request by probonopd**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# QMediathekView [![Build Status](https://travis-ci.org/adamreichold/QMediathekView.svg?branch=master)](https://travis-ci.org/adamreichold/QMediathekView)
+
 _QMediathekView_ is an alternative Qt-based front-end for the database maintained by the [MediathekView](http://zdfmediathk.sourceforge.net/) project. It has fewer features than the Java-based original, but should also consume less resources.
 
+![Screenshot](https://user-images.githubusercontent.com/2480569/50729982-2f8ca580-113b-11e9-8602-250df8289437.png)
```
Comment (adamreichold) [inline README.md]: The screenshot is missing the SVG icons for play and record? I guess this is due to the SVG icon engine missing? Or do I need to package those icons into the AppImage explicitly?

Comment (adamreichold) [reply README.md]: (The AppImage I just released here using `iconengines/` instead of `imageformats/` seems to work?)

Comment (probonopd) [reply README.md]: Good catch, don't know yet

Comment (adamreichold) [reply README.md]: I think this was due to not installing `qt59svg` in the build environment...

Comment (probonopd) [reply README.md]: Great catch again!

Comment (adamreichold) [reply README.md]: An updated build was published. Could you update the screenshot? I'll merge this and #11 tomorrow then.

**Pull request merged**